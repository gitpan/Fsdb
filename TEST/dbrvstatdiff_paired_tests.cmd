prog='dbrvstatdiff'
args='-h "=0" mean stddev n copylast_mean copylast_stddev copylast_n'
cmp='diff -cb '
altcmp='dbfilediff --quiet -E -N test_diff --exit '
altcmp_needs_input_flags=true
