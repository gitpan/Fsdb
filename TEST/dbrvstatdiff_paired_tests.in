#fsdb case mean stddev pct_rsd conf_range conf_low conf_high conf_pct sum sum_squared min max n copylast_mean copylast_stddev copylast_n
b	1.98	0.083666	4.2256	0.10387	1.8761	2.0839	0.95	9.9	19.63	1.9	2.1	5	1.02	0.083666	5
#  | dbmultistats -k case value
#   | dbcolcopylast mean stddev n
#   | dbrow _case eq "b"
