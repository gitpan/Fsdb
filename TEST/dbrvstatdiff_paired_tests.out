#fsdb case mean stddev pct_rsd conf_range conf_low conf_high conf_pct sum sum_squared min max n copylast_mean copylast_stddev copylast_n diff diff_pct diff_conf_half diff_conf_low diff_conf_high diff_conf_pct_half diff_conf_pct_low diff_conf_pct_high t_test t_test_result t_test_break t_test_break_pct
b	1.98	0.083666	4.2256	0.10387	1.8761	2.0839	0.95	9.9	19.63	1.9	2.1	5	1.02	0.083666	5	-0.96	-48.485	0.12202	-1.082	-0.83798	6.1627	-54.648	-42.322	-18.142	rejected	-1.082	-54.648
#  | dbmultistats -k case value
#   | dbcolcopylast mean stddev n
#   | dbrow _case eq "b"
#   | dbrvstatdiff -h =0 mean stddev n copylast_mean copylast_stddev copylast_n
